pkg/azuredisk/azuredisk_v2.go

```diff
@@ -368,19 +370,29 @@ func (d *DriverV2) StartControllersAndDieOnExit(ctx context.Context) {
 	// Leader controller manager should recover CRI if possible and clean them up before exiting.
 	go func() {
 		<-mgr.Elected()
+		var errors []error
+		ctx, workflow := azureutils.NewWorkflow(ctx)
+		defer func() {
+			if len(errors) == 0 {
+				return
+			}
+			errMsg := fmt.Sprintf("%v+", errors)
+			err := status.Error(codes.Internal, errMsg)
```

Comment:
Use `status.Errorf()` instead of `fmt.Sprintf()` and `status.Error()` separately.